include/roaring/portability.h

```diff
@@ -27,11 +27,10 @@
 #include <malloc.h> // this should never be needed but there are some reports that it is needed.
 #endif
 
-#if __SIZEOF_LONG_LONG__ != 8
+#if defined(__SIZEOF_LONG_LONG__) && __SIZEOF_LONG_LONG__ != 8
```

Comment:
Anyhow, I think that this check is only needed on GCC-like compilers due to the intrinsics. GCC has intrinsics for int, long and long long. If long long is only 4 bytes, then we have no intrinsic for 8-byte values... which occurs on some 32-bit systems. 

Still, for Visual Studio, I suspect that the same problem will occur... right? If you are compiling a 32-bit binary, you will be missing some 64-bit intrinsics, won't you?